### PR TITLE
[14.0][ADD] product_pricelist_item_button

### PR DESCRIPTION
**Pull request by rousseldenis**
Depends on:
- [ ] #910 

- [ ] Remove test-requirements.txt entry

Sélection_660

### CODE REVIEW

File: product_pricelist_item_button/models/product_pricelist_item.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import _, models
+
+
+class ProductPricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    def show_rules(self):
```
Comment (legalsylvain) [inline product_pricelist_item_button/models/product_pricelist_item.py]: Maybe show_items could be a better name

Comment (rousseldenis) [reply product_pricelist_item_button/models/product_pricelist_item.py]: I've followed Odoo display names :-)

Comment (rousseldenis) [reply product_pricelist_item_button/models/product_pricelist_item.py]: But I can change

Comment (legalsylvain) [reply product_pricelist_item_button/models/product_pricelist_item.py]: matter of taste. i usually use technical name for technical name function. but really not a blocking point !

File: product_pricelist_item_button/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "name": "Product Pricelist Item Button",
+    "summary": """
+        Allows to access to pricelist items tree view to get search features""",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "maintainers": ["rousseldenis"],
+    "website": "https://github.com/OCA/product-attribute",
+    "depends": [
+        "product",
```
Comment (dalonsod) [inline product_pricelist_item_button/__manifest__.py:14]: Unnecessary dependency (already included in `product_pricelist_button_box`)
```suggestion
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Locally tested OK. See comment, and test should be provided.

Non-blocking suggestions:
* Maybe a useful `active_item_ids_count` should be added (and used in the link button), indicating the current items active by date, with the related filter enabled by default. Anyway, I don't really know if such thing is easy to achieve...
* Hiding the current tab with rules should be considered, because these data view is replaced by the new button. It should take the advantage of a faster load of `product.pricelist` form when item list becomes too long.

@rousseldenis I'm interested in porting this addon to v13, so I can help with this PR if you need help to finish it

**Pull request closed without merging**